docs/modules/ROOT/pages/type-definitions/schema-configuration/type-configuration.adoc

```diff
@@ -0,0 +1,240 @@
+[[schema-configuration-type-configuration]]
+= Type Configuration
+
+When representing a Neo4j Node, a GraphQL Object type produces multiple operation fields in the `Query`, `Mutation`, and `Subscription` types. 
+For example:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String 
+    length: Int
+}
+----
+
+From these type definitions, the library generates the following operation fields:
+
+**Query**:
+
+    * `movies`
+    * `moviesAggregate`
+    * `moviesConnection`
+
+**Mutation**:
+
+    * `createMovies`
+    * `deleteMovies`
+    * `updateMovies`
+
+**Subscription**:
+
+    * `movieCreated`
+    * `movieUpdated`
+    * `movieDeleted`
+.
+
+This section explains how to reduce the operation fields produced using the directives `@query`, `@mutation`, and `@subscription`.
+
+== `@query`
+
+This directive is used to limit the availability of query operations in the library.
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+directive @query(read: Boolean! = true, aggregate: Boolean! = true) on OBJECT | SCHEMA
```

Comment:
This is defaulted to `false`, however, in version 3.x aggregations are generated if no `@query` is present. The breaking change comes when aggregations are not generated even when `@query` is not used

```suggestion
directive @query(read: Boolean! = true, aggregate: Boolean! = false) on OBJECT | SCHEMA
```

Reply:
Ouch, nice catch!